front/src/components/drawers/menuDrawer/MenuDrawer.tsx

```diff
@@ -12,12 +12,25 @@ enum Theme {
     Dark = "dark",
 }
 
-function MenuDrawer({ isOpen, onClose }: Props) {
-    const [theme, setTheme] = useState<Theme>(Theme.Light);
+const THEME_KEY = "theme";
+
+function getLocalStoredTheme(): Theme {
+    const storedTheme = localStorage.getItem(THEME_KEY);
+    if (storedTheme === null) {
+        return Theme.Light;
+    }
+    if (storedTheme === Theme.Light) {
+        return Theme.Light;
+    }
+    return Theme.Dark;
+}
```

Comment:
```suggestion
function getLocalStoredTheme(): Theme {
    return localStorage.getItem(THEME_KEY) === Theme.Dark ? Theme.Dark : Theme.Light
}
```

If we plan to add additional themes in the future, we should change this to a `swtich` statement though.